framework/include/meshgenerators/RefineBlockGenerator.h

```diff
@@ -37,6 +37,9 @@ class RefineBlockGenerator : public MeshGenerator
   /// Toggles whether neighboring level one elements should be refined or not. Defaults to true.
   const bool _enable_neighbor_refinement;
 
+  /// maximum element size over which elements are refined
+  const Real _max_element_size;
```

Comment:
I would rename `size` to `volume`. `size` could be related to `hmin()` or `hmax()` for example, so there's some ambiguity

Reply:
will do